### PR TITLE
front: add logo when train is projected

### LINKED ISSUE #9191: Add an icon when a train is projected
### Who would benefit from this feature?

Both

### What is this feature about?

mockup: 
image

### PR DESCRIPTION
**Pull request by theocrsb**
This PR needs [this ticket](https://github.com/OpenRailAssociation/osrd-ui/issues/606) to be finished
closes #9191

### CODE REVIEW

File: front/src/styles/scss/applications/operationalStudies/_scenario.scss [hunk 1]
```diff
@@ -332,6 +332,18 @@
...
+          display: flex;
+          align-items: center;
+          .train-projected {
+            height: 1.25rem;
+            width: 1.625rem;
+            position: relative;
+            svg {
+              position: absolute;
+              top: -0.125rem;
+              left: -0.125rem;
```
Comment (Math-R) [inline front/src/styles/scss/applications/operationalStudies/_scenario.scss]: in some case use negative values for positionning an absolute element could lead the element to expand his  size, tryc to set it to 0 and then apply a transformation

Comment (theocrsb) [reply front/src/styles/scss/applications/operationalStudies/_scenario.scss]: Done

**Review by Math-R — APPROVED** [review]
lgtm

**Review by RomainValls — APPROVED** [review]
Lgtm, tested 👍

**Pull request merged**